tox.ini

```diff
@@ -15,3 +16,5 @@ commands =
   flake8 xsnippet_api/ tests/
 # check that unittests are passed
   py.test --cov xsnippet_api/ --cov-append tests/
+# check the swagger definition of the API
+  swagger-flex -s contrib/swagger.yaml
```

Comment:
Hm.. The reason I put `setup.py check` and `flake8` check under `[testenv]` declaration is that we usually want to perform those checks for each Python interpreter. Why? Because the results may be different in those cases (`setup.py` may have if branches, and `flake8`, if I'm not mistaken, may work differently for each interpreter).

What about `swagger-flex`? Does it performs interpreter-independent check? If so, I'd prefer to move it into separate `tox` env. What do you think?